Tests/test_themes.py

```
@@ -1,109 +1,114 @@
-            button_id = dpg.add_button(label="Test Button")
+    assert Themes.DEFAULT in ThemeManager._item_themes[button_id]
+    assert Themes.ERROR in ThemeManager._item_themes[button_id]
+    assert len(ThemeManager._item_themes[button_id]) == 2
 
-        ThemeManager.apply_theme(button_id, Themes.ERROR)
 
-        theme_key = (Themes.ERROR,)
-        assert theme_key in ThemeManager._created_themes
+def test_remove_theme(env):
+    """
+    Проверяет удаление темы у элемента.
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `у` (CYRILLIC SMALL LETTER U). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF002)